tidy3d/web/cli/app.py

```diff
@@ -16,8 +16,13 @@
 from ..core.environment import Env
 from .develop.index import develop
 
-if not os.path.exists(TIDY3D_DIR):
-    os.mkdir(TIDY3D_DIR)
+
+# Prevent race condition on threads
+try:
+    os.makedirs(TIDY3D_DIR)
+except OSError as e:
+    if e.errno != errno.EEXIST:
+        raise  # Reraise if failed for reasons other than existing already
```

Comment:
Thanks. This seems equivalent to `os.makedirs(TIDY3D_DIR, exist_ok=True)`. According to the link you shared, this version is only needed for python < 3.2, but we require >= 3.9, so let's just use the simple version. Could you update?

Reply:
Thanks for your reply! Sure, this should work as well